listenbrainz/webserver/static/js/src/listens/ListenCountCard.tsx

```diff
@@ -28,11 +26,16 @@ const ListenCountCard = (props: ListenCountCardProps) => {
     );
   } else {
     content = (
-      <p>
-        {isCurrentUser
-          ? "You have not listened to any songs so far"
-          : `${user.name} has not listened to any songs so far`}
-      </p>
+      <>
+        <p className="text-muted">
+          {isCurrentUser ? "Your" : `${user.name}'s`} listens count
+        </p>
+        <hr style={{ margin: "10px 0px" }} />
+        <div style={{ fontSize: "14px" }} className="text-muted">
+          {isCurrentUser ? "You haven't" : `${user.name} hasn't`} listened to
+          any songs.
```

Comment:
What do you think about this optimistic note? :)

```suggestion
          any songs yet.
```

Reply:
I think I like it :)